operators/minio-operator-rhmp/5.0.14/manifests/minio-operator-rhmp.clusterserviceversion.yaml

```diff
@@ -0,0 +1,785 @@
+            },
+            "labels": {
+              "app": "minio"
+            },
+            "name": "myminio",
+            "namespace": "minio-operator"
+          },
+          "spec": {
+            "certConfig": {},
+            "configuration": {
+              "name": "storage-configuration"
+            },
+            "env": [],
+            "externalCaCertSecret": [],
+            "externalCertSecret": [],
+            "externalClientCertSecrets": [],
+            "features": {
+              "bucketDNS": false,
+              "domains": {}
+            },
+            "image": "quay.io/minio/minio@sha256:ea2c70cda70860c7d10f04ce1df640d3abb995784cb7aeedef8d4baa53a5a113",
+            "imagePullSecret": {},
+            "mountPath": "/export",
+            "podManagementPolicy": "Parallel",
+            "pools": [
+              {
+                "containerSecurityContext": {},
+                "name": "pool-0",
+                "securityContext": {},
+                "servers": 4,
+                "volumeClaimTemplate": {
+                  "metadata": {
+                    "name": "data"
+                  },
+                  "spec": {
+                    "accessModes": [
+                      "ReadWriteOnce"
+                    ],
+                    "resources": {
+                      "requests": {
+                        "storage": "2Gi"
+                      }
+                    }
+                  }
+                },
+                "volumesPerServer": 2
+              }
+            ],
+            "priorityClassName": "",
+            "requestAutoCert": true,
+            "serviceAccountName": "",
+            "serviceMetadata": {
+              "consoleServiceAnnotations": {},
+              "consoleServiceLabels": {},
+              "minioServiceAnnotations": {},
+              "minioServiceLabels": {}
+            },
+            "subPath": "",
+            "users": [
+              {
+                "name": "storage-user"
+              }
+            ]
+          }
+        }
+      ]
+    capabilities: Full Lifecycle
+    categories: AI/Machine Learning, Big Data, Cloud Provider, Storage
+    description: |-
+      MinIO is a Kubernetes-native high performance object store with an
+      S3-compatible API. The MinIO Operator supports deploying MinIO Tenants
+      onto any Kubernetes.
+    features.operators.openshift.io/cnf: "false"
+    features.operators.openshift.io/cni: "false"
+    features.operators.openshift.io/csi: "false"
+    features.operators.openshift.io/disconnected: "true"
+    features.operators.openshift.io/fips-compliant: "true"
+    features.operators.openshift.io/proxy-aware: "true"
+    features.operators.openshift.io/tls-profiles: "false"
+    features.operators.openshift.io/token-auth-aws: "false"
+    features.operators.openshift.io/token-auth-azure: "false"
+    features.operators.openshift.io/token-auth-gcp: "false"
+    olm.skipRange: '<5.0.14'
```

Comment:
Based on the previous versions of how you release your operator, I do not think this is what you want. Normally you are use `replaces`, now that you are switching to be in a new catalog, you should remove this line and change your `replaces` that is normally at the end of the CSV to a `skips`

Reply:
More info on `skipRange` [here](https://olm.operatorframework.io/docs/concepts/olm-architecture/operator-catalog/creating-an-update-graph/#skiprange)

```
WARNING: As a consequence of using skipRange the skipped operator versions will be pruned from OLMs update graph and therefore will not be installable anymore by users with the spec.startingCSV property of Subscriptions. Use with caution. If you want to have direct (one hop) updates to this version from multiple previous versions but keep those previous versions available to users for install, always use skipRange in conjunction with replaces pointing to the immediate previous version of the operator version in question.
```